pkg/configuration/testdata/tnf_test_config.yml

```diff
@@ -20,3 +20,6 @@ certifiedoperatorinfo:
 acceptedKernelTaints:
   - module: "taint1"
   - module: "taint2"
+skipScalingTestPods:
```

Comment:
Pods? Shouldn't this be skipScalingTestDeploymentNames or skipScalingTestStatefulSetNames ?

Reply:
Yes originally I was targeting pods but then changed it to be statefulsets and deployments.  I'll fix.